datafusion/core/tests/sql/subqueries.rs

```diff
@@ -576,6 +576,8 @@ async fn support_order_by_correlated_columns() -> Result<()> {
     Ok(())
 }
 
+// TODO: issue https://github.com/apache/arrow-datafusion/issues/6263
```

Comment:
👍 so only 1 test gets failed on optimization rules?

Reply:
Yes

Reply:
Which rule fail this test ?

Reply:
> Which rule fail this test ?

decorrelate_where_in, you can see issue #6263

Reply:
I will fix this rule later.